app/models/mixins/aggregation_mixin.rb

```diff
@@ -4,9 +4,9 @@ module AggregationMixin
     virtual_aggregate :aggregate_cpu_speed,       :host_hardwares, :sum, :aggregate_cpu_speed
     virtual_aggregate :aggregate_cpu_total_cores, :host_hardwares, :sum, :cpu_total_cores
     virtual_aggregate :aggregate_disk_capacity,   :host_hardwares, :sum, :disk_capacity
-    virtual_aggregate :aggregate_memory,          :host_hardwares, :sum, :memory_mb
+    virtual_aggregate :aggregate_memory,          :host_hardwares, :sum, :ram_size_in_bytes
```

Comment:
sigh. yea, we need to revert

virtual aggregate should still work - don't want to change units